mysql-test/r/percona_heap_var.result

```diff
@@ -76,7 +75,7 @@ Name	t1
 Engine	MEMORY
 Version	10
 Row_format	Dynamic
-Rows	7
+Rows	0
```

Comment:
We discussed this on Slack, that it is a valid value, but why is this cached at 0 now?